automation/taskcluster/lib/tasks.py

```diff
@@ -561,6 +553,20 @@ def craft_function(signing_task_id, mozharness_task_id, variant, gecko_revision,
             )
         return craft_function
 
+    def craft_raptor_youtube_playback_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision, force_run_on_64_bit_device=False):
```

Comment:
We need to exclude tests for resolutions above 1080p on Moto G5, like we did in taskcluster via [bug 1558456](https://bugzilla.mozilla.org/show_bug.cgi?id=1558456). It looks like we could use `variant` to determine the target device. Perhaps we could add an optional argument to `_craft_raptor_task` that takes a list of additional command line options.

Reply:
Hi Dave,
I added the logic from bug 1558456 in this commit:
https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49